mambaSharedFramework/Pantos-Generic Playlist Parsing/PantosTag.swift

```diff
@@ -389,26 +389,26 @@ extension PantosTag: HLSTagDescriptor, Equatable {
             return GenericDictionaryTagValidator(tag: pantostag, dictionaryValueIdentifiers: [
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.bandwidthBPS, optional: false, expectedType: Int.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.programId, optional: true, expectedType: Int.self),
-                DictionaryTagValueIdentifierImpl(valueId: PantosValue.codecs, optional: true, expectedType: HLSCodecArray.self),
-                DictionaryTagValueIdentifierImpl(valueId: PantosValue.resolution, optional: true, expectedType: HLSResolution.self),
+                DictionaryTagValueIdentifierImpl(valueId: PantosValue.codecs, optional: true, expectedType: CodecValueTypeArray.self),
+                DictionaryTagValueIdentifierImpl(valueId: PantosValue.resolution, optional: true, expectedType: ResolutionValueType.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.audioGroup, optional: true, expectedType: String.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.videoGroup, optional: true, expectedType: String.self),
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.subtitlesGroup, optional: true, expectedType: String.self),
-                DictionaryTagValueIdentifierImpl(valueId: PantosValue.closedCaptionsGroup, optional: true, expectedType: HLSClosedCaptions.self)
+                DictionaryTagValueIdentifierImpl(valueId: PantosValue.closedCaptionsGroup, optional: true, expectedType: ClosedCaptionsValueType.self)
                 ])
             
         case .EXT_X_MEDIA:
             return GenericDictionaryTagValidator(tag: pantostag, dictionaryValueIdentifiers: [
                 DictionaryTagValueIdentifierImpl(valueId: PantosValue.uri, optional: true, expectedType: String.self),
-                DictionaryTagValueIdentifierImpl(valueId: PantosValue.type, optional: true, expectedType: HLSMediaType.self),
+                DictionaryTagValueIdentifierImpl(valueId: PantosValue.type, optional: true, expectedType: MediaType.self),
```

Comment:
Observation, in general `HLS...` is replaced by `Playlist...`. `MediaType` is one of the few exceptions.

Reply:
This one is OK IMO. `MediaType` is potentially usable outside the Playlist "domain".